### PR TITLE
Add function to cancel trustline updates requests

### PR DESCRIPTION
**Pull request by cducrest**
Only initiator of request can cancel it

### CODE REVIEW

File: contracts/CurrencyNetwork.sol [hunk 1]
```diff
@@ -348,6 +353,24 @@ contract CurrencyNetwork is CurrencyNetworkInterface, Authorizable, debtTracking
         );
     }
 
+    /**
+     * @notice `msg.sender` cancels a trustline update it initiated with _debtor
+     * @param _counterparty The other party of the trustline agreement
+     */
+    function cancelTrustlineUpdate(
```
Comment (berndbohmeier) [inline contracts/CurrencyNetwork.sol]: Maybe for consistency `cancelUpdateTrustline`?

File: contracts/CurrencyNetwork.sol [hunk 2]
```diff
@@ -348,6 +353,24 @@ contract CurrencyNetwork is CurrencyNetworkInterface, Authorizable, debtTracking
...
+        address _counterparty
+    )
+        external
+    {
+        require(! isNetworkFrozen, "The network is frozen; trustlines update cannot be canceled.");
+
+        TrustlineRequest memory trustlineRequest = _loadTrustlineRequest(msg.sender, _counterparty);
+        require(trustlineRequest.initiator == msg.sender, "No update request from msg.sender to _debtor.");
```
Comment (berndbohmeier) [inline contracts/CurrencyNetwork.sol]: Maybe also the other party should be able to cancel? (declining a trustline update)

File: contracts/CurrencyNetwork.sol (hunk 1 above)
Comment (cducrest) [reply contracts/CurrencyNetwork.sol]: I would rather not, we don't cancel update a trustline we cancel a trustline update.

File: contracts/CurrencyNetwork.sol (hunk 2 above)
Comment (cducrest) [reply contracts/CurrencyNetwork.sol]: Ok

File: contracts/CurrencyNetwork.sol (hunk 1 above)
Comment (berndbohmeier) [reply contracts/CurrencyNetwork.sol]: okay, leave it, but we change the other updateTrustline then to requestTrustlineUpdate

**Pull request merged**